src/posthog-core.ts

```diff
@@ -1831,9 +1831,17 @@ export class PostHog {
                   error,
               ])
             : {
-                  $exception_type: error.name,
-                  $exception_message: error.message,
                   $exception_level: 'error',
+                  $exception_list: [
+                      {
+                          type: error.name,
+                          value: error.message,
+                          mechanism: {
+                              handled: true,
+                              synthetic: false,
```

Comment:
Should this be `synthetic: true`? We don't know for certain if it's just a string

Reply:
captureException only takes an error as a param though, not string 🤔 - and in this case we can't do stack resolution either, so not sure how we'd handle the synthetic case.. `error.name` would be undefined as well.

Reply:
Won't that only apply to people using Typescript? I guess someone could give us a string but I'm not even sure how we'd handle that right now... Let's revisit it once we have a few more examples. This can be the happy path for now 👍